contrib/format-excel/README.md

```diff
@@ -0,0 +1,36 @@
+# Excel Format Plugin
+This plugin enables Drill to read Microsoft Excel files.  This format is best used with Excel files that do not have extensive formatting, however it will work with formatted files, by allowing you to define a region within the file where the data is.  
```

Comment:
Please remove additional spaces after dots... Not sure what formatting was used but they appear everywhere in .md file.